### PR TITLE
freewheel-ssp Bid Adapter: add schain in request params

### PR DESCRIPTION
**Pull request by xwang202**
## Type of change
- [x] Feature

## Description of change
add schain in request params

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@xwang202 schain seems to be causing a linting error in the way it is passed in the test file

**Review by patmmccann — CHANGES_REQUESTED** [review]

File: test/spec/modules/freewheel-sspBidAdapter_spec.js [hunk 1]
```diff
@@ -45,6 +58,11 @@ describe('freewheelSSP BidAdapter Test', () => {
       };
       expect(spec.isBidRequestValid(bid)).to.equal(false);
     });
+
+    it('should return a properly formatted request with schain defined', function () {
+      const bidRequests = spec.buildRequests(bid);
```
[the remark this answers is not in the document]
Comment (xwang202) [reply test/spec/modules/freewheel-sspBidAdapter_spec.js]: Hi @patmmccann ,  I updated again. Please check. Thanks.

[the remark this answers is not in the document]
Comment (xwang202) [reply test/spec/modules/freewheel-sspBidAdapter_spec.js]: @patmmccann The test has been updated. I think the new error are not caused by our test. Please double check. Thanks.

**Review by patmmccann — APPROVED** [review]

**Comment by patmmccann** [comment]
LGTM

**Pull request merged**